create.sql

```diff
@@ -557,7 +571,8 @@ CREATE TABLE step_f2 (
     exposure_time integer NOT NULL,
     filter identifier NOT NULL,
     lyot_wheel identifier NOT NULL,
-    disperser identifier NOT NULL
+    disperser identifier NOT NULL,
+    window_cover identifier NOT NULL
```

Comment:
I still trying to understand this project, but if this SQL table was automatically created with `pg_dump` where does the original `step_f2` come from?